### ISSUE TITLE
Import error trying to include in my project

### ISSUE THREAD

**Issue by GraysonNull** [opentok/web-components#36]
Hello, 

I'm trying to import the web components into my React project and I'm getting the following build error:

```Collecting page data  /node_modules/@vonage/video-publisher/video-publisher.js:1
import { VideoPublisher } from './src/VideoPublisher.js';

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1176:20)
    at Module._compile (node:internal/modules/cjs/loader:1218:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at mod.require (/node_modules/next/dist/server/require-hook.js:64:28)
    at require (node:internal/modules/cjs/helpers:110:18)
    at 9149 (/.next/server/pages/meetings/call/[id].js:1:2322)

> Build error occurred
```

I'm importing the module into a `.tsx` file simply by doing `import '@vonage/video-publisher/video-publisher.js';`

Node: 18.15.0
React: 18.2.0

Here's what my tsconfig compiler options look like
```"compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "checkJs": true,
    "removeComments": true,
    "skipLibCheck": true,
    "sourceMap": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "incremental": true,
    "downlevelIteration": true,
    "esModuleInterop": true,
    "extendedDiagnostics": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "typeRoots": [
      "./node_modules/@types",
      "./types"
    ]}
```

I've tried a lot of the things the internet has suggested but have had no luck. Any help you could provide on this would be helpful. Thanks!

**Comment by conshus** [comment]
Hi @GraysonNull 
> import '@vonage/video-publisher/video-publisher.js';

didn't work?

**Comment by conshus** [comment]
Looked more into this. Could you say what you tried?

Did you try adding `"type": "module"` to your package.json?

You can try looking at the angular example's tsconfig and comparing it to yours.

**Comment by GraysonNull** [comment]
Hi Dwane, yeah importing it how you mentioned and then trying to use the web component I was getting another error I can't remember exactly what it was. I ended up just going a different route and building some hooks based off of a now archived open source repo. High probability I had my wires crossed somewhere that was causing me problems and not the library itself. Happy to close this issue since I found a different path to success. 

Thanks!
